Portfolio mobile.html

```diff
@@ -71,6 +71,84 @@ <h3 class="support_head">Software Developer</h3>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <div class="techstack">
+                <div class="css">css</div>
+                <div><img src="./images/Rectangle%2070.png" alt="split"/></div>
+                <div class="html">html</div>
+                <div><img src="./images/Rectangle%2070.png" alt="split"/></div>
+                <div class="bootstrap">bootstrap</div>
+                <div><img src="./images/Rectangle%2070.png" alt="split"/></div>
+                <div class="ruby">Ruby</div>
+            </div>
+        <button type="button">See Project</button>
+    </section>
+
+    <section class="Project_work">
+        <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+        <h3 class="posted">Multi-Post Stories</h3>
+        <img id="ellipse21" src="./images/Ellipse%2021.png" alt="ellipse 21"/>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <div class="techstack">
+                <div class="css">css</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="html">html</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="bootstrap">bootstrap</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="ruby">Ruby</div>
+            </div>
+        <button type="button">See Project</button>
+    </section>
+
+    <section class="Project_work">
+        <img class="Placeholder" src="./images/Img%20Placeholder.png" alt="Image place holder"/>
+        <h3 class="posted">Multi-Post Stories</h3>
+        <img id="ellipse22" src="./images/Ellipse%2022" alt="ellipse 22"/>
+        <p class="text">A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+            <div class="techstack">
+                <div class="css">css</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="html">html</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="bootstrap">bootstrap</div>
+                <img src="./images/Rectangle%2070.png" alt="split"/>
+                <div class="ruby">Ruby</div>
+            </div>
+        <button type="button">See Project</button>
+    </section>
+
```

Comment:
- [ ] Kindly you should add the about myself section because it's the part of the project requirements as suggested by the previous reviewer .and this about section is where you will need to showcase your skills in the future

Reply:
## Optional
- [ ] Please address this as requested with previous reviewer before merging

Reply:
I cannot make my technologies into a list because they are side by side and I cannot undo my flexbox to change the direction to side by side and there is no about section in my template 4 in figma